profiles/include/Panel_4K_VideoPort.yaml

```diff
@@ -0,0 +1,142 @@
+    #dsTV_RESOLUTION_2160p60      = 0x100000   ///< 2160p60 Resolution
+    Supported_tv_resolutions_capabilities: 0x1443F3 # OR-ed value of 480i | 480p | 576p50 | 720p | 720p50 | 1080i | 1080p | 1080p24 | 1080i50 | 2160p30 | 2160p60
+    #dsHDRSTANDARD_NONE               = 0x0,   ///< When No video format is decoded
+    #dsHDRSTANDARD_HDR10              = 0x01,  ///< Video Format HDR
+    #dsHDRSTANDARD_HLG                = 0x02,  ///< Video Format HLG
+    #dsHDRSTANDARD_DolbyVision        = 0x04,  ///< Video Format Dolby Vision
+    #dsHDRSTANDARD_TechnicolorPrime   = 0x08,  ///< Video Format Techinicolor Prime
+    #dsHDRSTANDARD_HDR10PLUS          = 0x10,  ///< Video Format HDRPlus 
+    #dsHDRSTANDARD_SDR                = 0x20,  ///< Video Format SDR 
+    #dsHDRSTANDARD_Invalid            = 0x80   ///< When invalid value observed
+    # HDR Capabilities:
+    hdr_capabilities: 0x37 # OR-ed value of SDR | HDR10 | HDR10PLUS | DolbyVision | HLG
+    # HDCP Protocol Version:
+    #dsHDCP_VERSION_1X                = 00x0,  ///< HDCP Protocol version 1.x 
+    #dsHDCP_VERSION_2X                = 0x01 , ///< HDCP Protocol version 2.x 
+    hdcp_protocol_version: 0x01 #VERSION_2X
+    # Quantization Ranges:  
+    #dsDISPLAY_QUANTIZATIONRANGE_UNKNOWN = 0x0,   ///< Unknown quantization range  
+    #dsDISPLAY_QUANTIZATIONRANGE_LIMITED = 0x01,  ///< Limited quantization range  
+    #dsDISPLAY_QUANTIZATIONRANGE_FULL    = 0x02,  ///< Full quantization range 
+    quantization_ranges: 0x02 #FULL
+    # Matrix Coefficients
+    #dsDISPLAY_MATRIXCOEFFICIENT_UNKNOWN          = 0x00, ///< Unknown Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_BT_709           = 0x01, ///< ITU BT 709 Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_BT_470_2_BG      = 0x02, ///< ITU BT 470_2_BG Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_SMPTE_170M       = 0x03, ///< SMPTE 170M Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_XvYCC_709        = 0x04, ///< XvYCC_709 Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_eXvYCC_601       = 0x05, ///< XvYCC_601 Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_BT_2020_NCL      = 0x06, ///< ITU BT 2020 non constant luminance Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_BT_2020_CL       = 0x07, ///< ITU BT 2020 constant luminance Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_eDVI_FR_RGB      = 0x08, ///< eDVI Full Range RGB Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_eHDMI_RGB        = 0x09, ///< eHDMI RGB Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_eFCC             = 0x0A, ///< eFCC Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_eSMPTE_240M      = 0x0B, ///< eSMPTE 240M Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_eHDMI_FR_YCbCr   = 0x0C, ///< eHDMI Full Range YcbCr Matrix Coefficient
+    #dsDISPLAY_MATRIXCOEFFICIENT_MAX              = 0xFF  ///< Out of range
+    matrix_coefficients: 0x07 #BT_2020_CL
+
+# 8-Bit, 10-bit and 12 bit
+color_depth: 10 

```

Comment:
can we move to the top rather than the bottom of the file? To group global configuration together.

Reply:
Done